src/phoenix/datasets/experiments.py

```diff
@@ -184,36 +188,29 @@ async def async_run_experiment(test_case: TestCase) -> ExperimentRun:
         if payload is not None:
             resp = client.post(f"/v1/experiments/{experiment_id}/runs", json=jsonify(payload))
             resp.raise_for_status()
-    return Experiment(
+
+    experiment = Experiment(
         id=experiment_id,
         dataset_id=dataset.id,
         dataset_version_id=dataset.version_id,
     )
 
+    if evaluators:
+        for evaluator in evaluators:
+            _evaluate_experiment(experiment, evaluator, dataset.examples)
```

Comment:
this runs sequentially no? Wouldn't we want to maximize efficiency here by at least executing a few at a time?

Reply:
they run sequentially but are parallelized within each evaluator, it's trivial to combine the payloads though so I can do it that way